compiler/src/dotty/tools/dotc/semanticdb/ExtractSemanticDB.scala

```diff
@@ -429,7 +429,7 @@ class ExtractSemanticDB extends Phase:
         props |= SymbolInformation.Property.FINAL.value
       if sym.is(Sealed) then
         props |= SymbolInformation.Property.SEALED.value
-      if sym.isOneOf(GivenOrImplicit) then
```

Comment:
It might be better to keep `implicit` property for `given`s for backward compatibility, and remove it maybe in the next minor version release of Scala3?

Reply:
We tried that already, but it turned out that it was overall simplest the way it is now.